src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c

```diff
@@ -230,6 +230,7 @@ extern "C" {
             }
```

Comment:
The comparison above should be `if (result == 0)` instead of `TRUE`, because [`FreeLibrary`](https://learn.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-freelibrary) returns non-zero if the function succeeds, which implies the return value could be different from 1 (`TRUE`).

Reply:
Fixed. I also changed the logic below to return FALSE if the FreeLibrary call above failed